### PR TITLE
make use of PXR_OVERRIDE_PLUGINPATH_NAME everywhere

### PR DESCRIPTION
**Pull request by pmolodo**
PXR_OVERRIDE_PLUGINPATH_NAME is already used in some places, but it's coverage is spotty - make use of it everywhere, instead of hard-coding PXR_PLUGINPATH_NAME

### CODE REVIEW

File: test/lib/usd/translators/testUsdExportCustomConverter.template.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ class testUsdExportCustomConverter(unittest.TestCase):
     def setUpClass(cls):
         # In case tests are run in parallel, make sure we export to separate
         # directories:
-        plugin_path = os.getenv("PXR_PLUGINPATH_NAME")
+        plugin_path = os.getenv("@PXR_OVERRIDE_PLUGINPATH_NAME@")
```
Comment (seando-adsk) [inline test/lib/usd/translators/testUsdExportCustomConverter.template.py:35]: Thats a neat trick using the configure_file

**Review by seando-adsk — APPROVED** [review]
Everything looks good to me.

**Review by mattyjams — APPROVED** [review]
Looks good to me!

File: test/lib/usd/translators/testUsdExportCustomConverter.template.py (hunk 1 above)
Comment (pmolodo) [reply test/lib/usd/translators/testUsdExportCustomConverter.template.py:35]: Honestly, I wanted to find a way that DIDN'T have to use `configure_file`... but I couldn't think of any alternative.  At least this works!

**Pull request merged**